doc/API.md

```diff
@@ -336,10 +336,19 @@ struct {
 	unlockhash   types.UnlockHash  (string)
 	windowsize   types.BlockHeight (uint64)
 
+	acceptingcontracts bool
 	numcontracts       uint64
 	revenue            types.Currency (string)
 	storageremaining   int64
 	anticipatedrevenue types.Currency (string)
```

Comment:
clusters should be alphabetically sorted, previous PR screwed this up, can you fix it?